spec/system/comments/debates_spec.rb

```diff
@@ -239,12 +245,14 @@
     click_link "Reply"
 
     within "#js-comment-form-comment_#{comment.id}" do
-      fill_in "comment-body-comment_#{comment.id}", with: "It will be done next week."
+      fill_in "Leave your comment", with: "It will be done next week."
       click_button "Publish reply"
     end
 
     within "#comment_#{comment.id}" do
       expect(page).to have_content "It will be done next week."
+      expect(page).not_to have_css("img.initialjs-avatar[src$='initialjs-blank.png']")
+      expect(page).to have_css("img.initialjs-avatar", count: 2)
```

Comment:
Nice and simple spec! 👏

I am agree with you, it could create confusion to see that lines only at this spec, to treat this into a separate scenario will be clearer.

>Otherwise it would look strange to have these lines in this test but not in similar (identical 😌) tests for proposals, polls, topics, ...

Regarding the placement of the spec ... What about to place it in users_spec? At the end it seems more a users feature that a comments one ... Dont you think? 🤔 Maybe we could adapt a little bit the scenario description to make it fit better to this file, something like:

```
  scenario "Users initials display SVG avatars when loaded into the DOM", :js do
    login_as(create(:user))
    visit debate_path(debate)

    fill_in "Leave your comment", with: "I'm awesome"
    click_button "Publish comment"

    within ".comment", text: "I'm awesome" do
      expect(page).to have_css "img.initialjs-avatar[src^='data:image/svg']"
    end
  end
```

Reply:
> What about to place it in users_spec?

:+1: